mmaction/datasets/pipelines/loading.py

```diff
@@ -323,7 +323,8 @@ class DenseSampleFrames(SampleFrames):
         sample_range (int): Total sample range for dense sample.
             Default: 64.
         num_sample_positions (int): Number of sample start positions, Which is
-            only used in test mode. Default: 10.
+            only used in test mode. Default: 10. That is to say, by default,
```

Comment:
> Number of sample start positions,

> at least 10 crops for one input sample

The first one reads like it samples in time axis, while the second seem like it is on spatial. Which is correct?

BTW the argument name is also not very clear

Reply:
it samples in time axis. it is indeed not accurate using `10 crops` here.

Reply:
Maybe `10 clips` is more accurate.

Reply:
ya